content/1_General/Contests.mdx

```diff
@@ -85,7 +76,7 @@ Let me know if there's something else I should try to qualify for!
 	<Resource
 		source="TopCoder"
 		title="Open"
-		url="https://tco20.topcoder.com/competition-overview/algorithm/algorithm-ways"
+		url="https://www.topcoder.com/community/member-programs/topcoder-open"
```

Comment:
can you remove this one too? no longer held

Reply:
really? is this topcoder open 2023 not the same thing https://tco23.topcoder.com/

Reply:
https://codeforces.com/blog/entry/116814

Reply:
ah ok